modules/gdscript/gdscript_analyzer.cpp

```diff
@@ -966,6 +966,13 @@ void GDScriptAnalyzer::resolve_class_member(GDScriptParser::ClassNode *p_class,
 
 				member.m_enum->set_datatype(enum_type);
 
+				Dictionary dictionary;
+				for (int j = 0; j < member.m_enum->values.size(); j++) {
+					// Needs to be string because Variant::get will convert to String.
+					dictionary[String(member.m_enum->values[j].identifier->name)] = member.m_enum->values[j].value;
+				}
```

Comment:
I believe this can be done in the same loop that fills the `enum_values` vector to avoid another iteration.

Reply:
Done.